src/components/AuthWrapper.vue

```diff
@@ -1,18 +1,25 @@
 <template>
-  <div class="main-full-height w-full flex justify-center items-center">
-    <div
-      class="bg-slate-200 rounded-lg shadow border border-gray-700 mx-auto inline-block w-96"
-    >
+  <div
+    class="bg-center bg-cover"
+    style="
+      background-image: url('https://i.pinimg.com/originals/b2/fe/91/b2fe91505525ba942ae7d85a2cd7a6d4.gif');
+    "
+  >
+    <div class="main-full-height w-full flex justify-center items-center">
       <div
-        class="flex p-6 space-y-4 md:space-y-6 sm:p-8 flex-auto flex-wrap justify-center justify-items-center content-center"
+        class="bg-slate-200 rounded-lg shadow border border-gray-700 mx-auto inline-block w-96"
```

Comment:
It seems it should look better now on small-screen devices but it keeps reaching the sides of the device. I suggest going back to previous styles and making the card more responsive on small-screen devices.

![deploy-preview-51--doitfy netlify app_auth_forgot-password(iPhone 12 Pro)](https://user-images.githubusercontent.com/87602505/195020773-98f65b29-1c2f-4f43-9c9e-8884f9266192.png)

Reply:
Btw there's an issue of responsiveness with the footer too.